### DISCUSSION TITLE
Provider Query

### DISCUSSION THREAD

**Discussion by asad-khan2000** [openmobilityfoundation/mobility-data-specification#612] [Technical]
I work for a micro-mobility company. I have the necessary URL and provider tag, provided by Joyride. I was wondering what software can I use to visualize and further analyze the data. Thank you.

**Comment by schnuerle** [comment]
Hi let me see if I can help you.  So you have access to MDS API endpoints provided by Joyride, and you have the access tokens/credentials to view the authenticated feeds.  Do you work for a different micromobility company?

If you have a feed, say /trips within Provider, then there are a few ways you can make sense of the data. Some agencies program scripts themselves to turn the JSON data into CSV or put it into a database. From there you can use tools like Power BI on a Windows desktop, QGIS or ESRI products. There are some open source [data processing tools](https://github.com/openmobilityfoundation/privacy-committee/blob/main/products/state-of-the-practice.md#data-processing-aggregation-and-anonymization) and [reference implementations](https://github.com/openmobilityfoundation/mds-core/wiki) if you are up for that. Others use third party solutions like Ride Report or Stae, or hire consultants.
